kubernetes-cluster-kubeone/docs/setup-demo.md

```diff
@@ -1,51 +1,60 @@
 ## Setup a Kubernetes Cluster for KSD using Hetzner Cloud
 
-This guide is to be ready and test KSD easily.
+This document will walk you through the steps required to set up a Kubernetes cluster in the Hetzner Cloud for use with KSD.
 
 ### IMPORTANT:
 
-In order to be able to follow all the steps, you would need a Hetzner Api token, to get this you will need: 
-- Sign in into the Hetzner Cloud Console 
-- choose a Project
-- go to Security → API Tokens
-- generate a new token.
+To follow this guide, you will need a Hetzner API token.  To generate this you will need to: 
+- Sign in into the [Hetzner Cloud Console](https://console.hetzner.cloud/)
+- Select your project
+- Go to Security → API Tokens
+- Generate a new token
```

Comment:
Don't we just use the one in 1password?

Reply:
To answer the first question, I was messing around with forking in the VSCode interface, & mucked up the branch.  Apologies!

As for the second, I believe this documentation is meant to be customer-facing at some point, so no need for the 1password stuff.

Reply:
Good point